.eslintrc.js

```diff
@@ -5,23 +5,28 @@ module.exports = {
         'es6': true
     },
     'extends': [
-        'eslint:recommended', "plugin:node/recommended"
+        'eslint:recommended', "plugin:node/recommended", 'plugin:react/recommended'
     ],
     'globals': {
         // The globals that (1) are accessed but not defined within many of our
         // files, (2) are certainly defined, and (3) we would like to use
         // without explicitly specifying them (using a comment) inside of our
         // files.
-        '__filename': false
+        '__filename': false,
+        'JitsiMeetExternalAPI': true
```

Comment:
Usually we are defining this type of globals in the files where they are used with comment in the beginning of the file:
`/* global JitsiMeetExternalAPI */`